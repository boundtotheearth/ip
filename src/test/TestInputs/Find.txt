todo read book
todo read novel
find book
list
