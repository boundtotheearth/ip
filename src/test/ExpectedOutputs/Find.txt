Added: [T][✘] read book
Added: [T][✘] read novel
Here are the matching tasks in your list:
1. [T][✘] read book

1. [T][✘] read book
2. [T][✘] read novel
